src/pages/media.js

```diff
@@ -13,10 +14,34 @@ export default () => {
         isLoading: isLoadingPodcasts,
     } = usePodcasts();
 
+    const [showContent, setShowContent] = useState('All');
+    const [activeItem, setActiveItem] = useState('All');
```

Comment:
Having two state vars for this is redundant, can we just have one? `[activeItem, setActiveItem]` and use that?

Reply:
Thanks for pointing that out!